IBPSA/Fluid/Geothermal/Aquifer/MultiWell.mo

```diff
@@ -0,0 +1,419 @@
+within IBPSA.Fluid.Geothermal.Aquifer;
+model MultiWell "Model of a single well for aquifer thermal energy storage"
+ replaceable package Medium = Modelica.Media.Interfaces.PartialMedium "Medium in the component" annotation (choices(
+        choice(redeclare package Medium = IBPSA.Media.Water "Water")));
+  parameter Integer nVol(min=1)=10 "Number of control volumes used in discretization" annotation (
+      Dialog(group="Subsurface"));
+  parameter Modelica.Units.SI.Height h=200 "Aquifer thickness";
+  parameter Real phi=0.2 "Reservoir porosity";
+  parameter Real nCoo=1 "Number of cold wells";
+  parameter Real nHot=1 "Number of warm wells";
+  parameter Modelica.Units.SI.Radius r_wb=0.1 "Wellbore radius" annotation (
+      Dialog(group="Subsurface"));
+  parameter Modelica.Units.SI.Radius r_max=2400 "Domain radius" annotation (
+      Dialog(group="Subsurface"));
+  parameter Real griFac(min=1) = 1.15 "Grid factor for spacing" annotation (
+      Dialog(group="Subsurface"));
+  parameter Modelica.Units.SI.Temperature TCoo_start=283.15
+     "Initial temperature of cold well" annotation (
+      Dialog(group="Subsurface"));
+  parameter Modelica.Units.SI.Temperature THot_start=TCoo_start
+     "Initial temperature of warm well" annotation (
+      Dialog(group="Subsurface"));
+  parameter Modelica.Units.SI.Temperature TGroCoo=285.15
+     "Undisturbed ground temperature (cold well)" annotation (
+      Dialog(group="Properties of ground"));
```

Comment:
What is the rationale for using an initial temperature for the well that is different from the undisturbed ground? Shouldn't they be the same?

Reply:
I assume that in some cases the user might be interested in performing a seasonal simulation starting from predefined initial temperatures in the aquifer. For example, one may carry out a winter simulation, and assume that the warm aquifer is fully charged at 30°C. However, the undisturbed ground temperature would still be at around 10°C.